src/version.c

```diff
@@ -96,13 +96,12 @@ int get_current_version(char *path_prefix)
 	FILE *file;
 	int v = -1;
 	char *buildstamp;
-	char *src, *dest;
 
-	string_or_die(&buildstamp, "%s/usr/lib/os-release", path_prefix);
```

Comment:
If you want to go this route, I would prefer seeing this change pushed in later, especially because it would tied to a format bump release.